internal/impl/openai/chat_processor.go

```diff
@@ -51,8 +51,24 @@ const (
 	ocpFieldSchemaRegistryNamePrefix      = "name_prefix"
 	ocpFieldSchemaRegistryURL             = "url"
 	ocpFieldSchemaRegistryTLS             = "tls"
+	// Tool options
+	ocpFieldTool                     = "tools"
```

Comment:
I wonder if this should be `ocpFieldTools`

Reply:
done